kscale/web/utils.py

```diff
@@ -13,7 +14,7 @@ def get_api_root() -> str:
     Returns:
         The base URL for the K-Scale Store API.
     """
-    return os.getenv("KSCALE_API_ROOT", "https://api.kscale.dev")
+    return os.getenv("KSCALE_API_ROOT", "https://api.kscale.dev/")
```

Comment:
Hmm please revert this, if it is causing a bug then that means you are missing a leading slash somewhere else. This will be annoying if the user wants to override this environment variable and it's missing a trailing slash